CONTRIBUTING.md

```diff
@@ -49,5 +49,14 @@ We use the Developer Certificate of Origin (DCO) as a additional safeguard for t
 
 Use your real name (sorry, no pseudonyms or anonymous contributions). If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
 
+### AI Policy
+
+We consider AI tools that help programming just one more tool that, if used judiciously, can help
+developers learn to code.  Volunteers using such AI tools is permitted to get good ideas, but you
```

Comment:
Maybe "Volunteers using such AI tools are permitted to get good ideas" or "Using such AI tools by volunteers is permitted to get good ideas"

Reply:
I went with the first.